extra-web/proton-bridge/autobuild/prepare

```diff
@@ -0,0 +1,2 @@
+export HOME=/root
```

Comment:
This shouldn't have been necessary, please hold off for the Autobuild3 topic to merge.